Quarterly HTAN Manual review; Phase 2 updates

jen-dfci opened the pull request:
Updates to text and images to reflect:
1. Phase 1 Synapse Wiki shut down.
2. New tool for metadata submission.

Additional text or format changes to improve clarity.

Reflects response to PR# 106 and PR#107, but additional changes will be needed once Phase 2 data submissions can start/new data model and tool are available.

aclayton555 commented on data_model/overview.md:
Do you want to include links to the data model repos?

jen-dfci replied:
that's a good idea.  just checking:
phase 1 link is: https://github.com/ncihtan/data-models
phase 2 link is: https://github.com/ncihtan/htan-linkml
@aclayton555 is that correct?

jen-dfci commented on addtnl_info/tool_protocol.md:
@alexeliotlash please see above -- does the tool curation form need to be updated?  
@inodb I believe you may use the results from this form, so also tagging you.

alexeliotlash replied:
I think we can remove the link to the tool curation form.  For the past couple of years, I've been using my pubmed search results to send new HTAN tools, when they pop up in the published or pre-print literature, to onur and ino.  Looks like the last time someone used the form was in 2022.  @inodb do you agree?

aclayton555 commented on data_submission/overview.md:
"the HTAN Network's repositories" 
HTAN Network is redundant. Suggest just saying "to HTAN repositories"

jen-dfci commented:
@aclayton555 Thank you again for your feedback.  The following changes were made:
1. Links to data model repos added to data model overview page. (with buttons! )
2. Changed "the HTAN Network's repositories" to "to HTAN repositories" in data submission overview.
3. removed the Tool curation form link and updated language on the Tool and Protocol Submission page.  (Also removed the Tool Curation page from the Synapse Wiki. And updated other Synapse wiki page to point to manual for info on submitting Tools.)

I think it is ready to go, but let me know what you think/if other changes are needed.
